code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,322 @@
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.reagents.add_reagent("regen_jelly", 50)
+	else
+		var/datum/reagents/R = new/datum/reagents(50)
+		R.add_reagent("regen_jelly", 50)
+		var/datum/effect_system/smoke_spread/chem/smoke = new
+		smoke.set_up(R, 1, get_turf(A))
+		smoke.start()
+
+/obj/item/slimecross/destabilized/blue
+	colour = "blue"
+
+/obj/item/slimecross/destabilized/blue/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/datum/reagents/R = new/datum/reagents(50)
+	R.add_reagent("water", 50)
+	R.reaction(get_turf(A), TOUCH, 50)
+	qdel(R) //Reaction doesnt use up the reagents
+	if(iscarbon(A))
+		var/mob/living/carbon/C = A
+		C.slip(100, src, FALSE, 20, FALSE)
+
+/obj/item/slimecross/destabilized/metal
+	colour = "metal"
+
+/obj/item/slimecross/destabilized/metal/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		to_chat(L, "<span class='warning'>You suddenly find yourself trapped in a dark cage!</span>")
+		L.apply_status_effect(/datum/status_effect/cageTrapped)
+	else
+		new /obj/structure/metalCage(get_turf(A))
+
+/obj/item/slimecross/destabilized/yellow
+	colour = "yellow"
+
+/obj/item/slimecross/destabilized/yellow/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.electrocute_act(rand(5,20), src, 1, 1)
+		playsound(L, "sparks", 50, 1)
+	else
+		empulse(get_turf(A), 1, 1)
+
+/obj/item/slimecross/destabilized/darkpurple
+	colour = "dark purple"
+
+/obj/item/slimecross/destabilized/darkpurple/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.fire_stacks += 5
+		L.reagents.add_reagent("plasma", 10)
+	else
+		var/datum/reagents/R = new/datum/reagents(10)
+		R.add_reagent("plasma", 10)
+		var/datum/effect_system/smoke_spread/chem/smoke = new
+		smoke.set_up(R, 1, get_turf(A))
+		smoke.start()
+
+/obj/item/slimecross/destabilized/darkblue
+	colour = "dark blue"
+
+/obj/item/slimecross/destabilized/darkblue/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.apply_status_effect(/datum/status_effect/freon/destabilized)
+
+/obj/item/slimecross/destabilized/silver
+	colour = "silver"
+
+/obj/item/slimecross/destabilized/silver/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.nutrition = NUTRITION_LEVEL_FAT + 1
+		to_chat(L, "<span class='warning'>You suddenly feel gorged!</span>")
+
+/obj/item/slimecross/destabilized/bluespace
+	colour = "bluespace"
+
+/obj/item/slimecross/destabilized/bluespace/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		do_teleport(A, get_turf(thrower), 1)
+	if(isobj(A))
+		var/obj/O = A
+		if(!O.anchored)
+			do_teleport(O, get_turf(thrower), 1)
+
+/obj/item/slimecross/destabilized/sepia
+	colour = "sepia"
+
+/obj/item/slimecross/destabilized/sepia/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/datum/proximity_monitor/advanced/timestop/TS = new()
+		TS.host = A
+		TS.freeze_atom(A)
```

Comment:
(better than) throwable taser bolt?

Reply:
It _is_ single use and (hopefully) somewhat time-consuming to obtain. With the progression away from stuns to stamina damage, though, it's definitely contest-able for a change.

Reply:
The timestop could probably be changed to a thing that massively slows the target

Reply:
Time stop is still a thing with spells, as well as other xenobio things, so I'm not exactly sure what's so controversial about this case in particular

Reply:
It's basically a single-use thrown lightning spell (y'know, the one that devils can give you if you wish for magic), which I think is okay to have in the game. Reinforced bolas (from a low-TC-cost throwing item kit) weren't used much back when they stunned people, so this should be fine.